ssn/rdf/ontology/core/sosa-actuation.ttl

```diff
@@ -250,11 +247,14 @@ sosa:startTime
   schema:domainIncludes sosa:ActuationCollection ;
   rdfs:isDefinedBy sosa-common: .
 
-sosa:resultTime 
+sosa:endTime 
   schema:domainIncludes sosa:Actuation ;
   schema:domainIncludes sosa:ActuationCollection ;
   rdfs:isDefinedBy sosa-common: .
 
+sosa:resultTime 
+  rdfs:isDefinedBy sosa-common: .
```

Comment:
We avoided `rdfs:domain` and `rdfs:range` entirely in the core ontology. 
We use `schema:domainIncludes` and `schema:rangeIncludes` in the SOSA modules. 
We use guarded constraints in the SSN modules. 

For `resultTIme`, in sosa-common.ttl see
```
sosa:resultTime 
  a owl:DatatypeProperty ;
  rdfs:label "result time"@en ;
  skos:definition """
  The instant of time when the Execution was completed, i.e. the Actuation or Observation or Sampling  
  """@en ;
  rdfs:comment "The value would usually be encoded using xsd:dateTime xsd:date xsd:gYearMonth xsd:gYear"@en ;
  schema:domainIncludes sosa:Execution ;
  rdfs:isDefinedBy sosa-common: .
```

Since subclassing is not seen by the SOSA modules, I guess we should add matching axioms for the subclasses of Execution (Actuation, Observation, Sampling) and System (Actuator, Sensor, Sampler) in sosa-actuation.ttl, sosa-observation.ttl, sosa-sampling.ttl . 

I'll make a separate branch and PR to check domain/range and specialization consistency.